### DISCUSSION TITLE
Quickstart error

### DISCUSSION THREAD

**Discussion by ghost** [egovernments/Digit-Core#113] [Local Governance]
I tried to install DIGIT to get a sense of the system requirements using k3s with a master and two nodes in a local environment. However, I get the following validating data error when I run the setup command:

-- Error message below --
Whola!, That's great... Sit back and wait for the deployment to complete in about 10 min
2023/06/02 06:28:08 Helm Directory - /home/shuller/DIGIT-DevOps/deploy-as-code/helm
2023/06/02 06:28:08 ------------------------------------ DEPLOYING CLUSTER CONFIGS ------------------------------------
/home/shuller/DIGIT-DevOps/deploy-as-code/helm/charts/cluster-configs
2023/06/02 06:28:08 helm template -f /home/shuller/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config.yaml --output-dir /tmp/helm-510347594 -f /home/shuller/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config-secrets.yaml .
2023/06/02 06:28:10 exit status 1: error: error validating "ingress/root-ingress.yaml": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0]): missing required field "pathType" in io.k8s.api.networking.v1.HTTPIngressPath]; if you choose to ignore these errors, turn validation off with --validate=false

panic: exit status 1: error: error validating "ingress/root-ingress.yaml": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0]): missing required field "pathType" in io.k8s.api.networking.v1.HTTPIngressPath]; if you choose to ignore these errors, turn validation off with --validate=false

goroutine 1 [running]:
log.Panicln(0xc000101790, 0x1, 0x1)
        /usr/lib/go-1.13/src/log/log.go:352 +0xac
[github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.execCommandRaw(0x92a0f0](http://github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.execCommandRaw(0x92a0f0), 0x1e, 0xc00019e000, 0x2d, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer/deployer.go:252 +0x316
[github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.deployClusterConfigs(0xc00008fb00](http://github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.deployClusterConfigs(0xc00008fb00), 0xc00002e360, 0x2e, 0xc000085440, 0x52, 0xc0000854a0, 0x5a)
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer/deployer.go:161 +0x757
[github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.DeployCharts(0x91ee6c](http://github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer.DeployCharts(0x91ee6c), 0x7, 0x7fffb17b76b6, 0x2e4, 0x7fffb17b76a4, 0x11, 0x1)
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/pkg/cmd/deployer/deployer.go:25 +0x18bd
[github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd.glob..func2(0xcd24e0](http://github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd.glob..func2(0xcd24e0), 0xc00006c740, 0x1, 0x4)
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd/deploy.go:51 +0x9b
[github.com/spf13/cobra.(*Command).execute(0xcd24e0](http://github.com/spf13/cobra.(*Command).execute(0xcd24e0), 0xc00006c6c0, 0x4, 0x4, 0xcd24e0, 0xc00006c6c0)
        /root/go/pkg/mod/[github.com/spf13/cobra@v0.0.5/command.go:830](http://github.com/spf13/cobra@v0.0.5/command.go:830) +0x2aa
[github.com/spf13/cobra.(*Command).ExecuteC(0xcd2760](http://github.com/spf13/cobra.(*Command).ExecuteC(0xcd2760), 0x43d07a, 0xc96920, 0xc000000180)
        /root/go/pkg/mod/[github.com/spf13/cobra@v0.0.5/command.go:914](http://github.com/spf13/cobra@v0.0.5/command.go:914) +0x2fb
[github.com/spf13/cobra.(*Command).Execute(..](http://github.com/spf13/cobra.(*Command).Execute(..).)
        /root/go/pkg/mod/[github.com/spf13/cobra@v0.0.5/command.go:864](http://github.com/spf13/cobra@v0.0.5/command.go:864)
[github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd.Execute()](http://github.com/egovernments/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd.Execute())
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/cmd/root.go:48 +0x31
main.main()
        /home/shuller/DIGIT-DevOps/deploy-as-code/egov-deployer/main.go:21 +0x20
exit status 2
2023/06/02 06:28:10 cmd.Run() failed with exit status 1
exit status 1

**Comment by talele08** [comment]
Please check the kubernetes version that you are using. For version above kubernetes 1.20 support for  v1beta API is not there. Please use helm charts from https://github.com/egovernments/DIGIT-DevOps/tree/DIGIT-Bootcamp this branch if you are using kubernetes version higher than 1.20
